### PR TITLE
Fix components path help message.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Fix components path help message.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/dapr/issues/1619
## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -245,7 +245,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
-	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ./components.")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ./.dapr/components.")
```
Comment (pruthvidhodda) [inline cmd/run.go:248]: ./.dapr/components. ==> ~/.dapr/components.

Comment (vinayada1) [reply cmd/run.go:248]: This needs to be %USERPROFILE%\.dapr\components for Windows

Comment (artursouza) [reply cmd/run.go]: Addressed.

File: cmd/run.go [hunk 2]
```diff
@@ -245,7 +245,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
-	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ./components.")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ~/.dapr/components or %USERPROFILE%\\.dapr\\components")
```
Comment (pruthvidhodda) [inline cmd/run.go:248]: Not an issue but to be more clear we can say :

"Default is ~/.dapr/components for linux and %USERPROFILE%\\.dapr\\components for windows"

Comment (artursouza) [reply cmd/run.go:248]: I thought about it but then we would need to explain macOS too.

**Pull request merged**